lib/rucio/tests/test_meta.py

```diff
@@ -83,3 +88,63 @@ def xtest_add_value_to_bad_key(self):
     def xtest_add_key_mysql(self):
         """ META (CORE): Add a new key to test conversions on MySQL"""
         self.meta_client.add_key('datatype', DIDType.FILE)
+
+    def xtest_insert_metadata(self):
```

Comment:
once you want to enable the unittests don't forget to remove the x in front of the name. only testcases starting with "test_..." will be executed.  "xtest_..." unittests are disabled.